lib/src/contextual-menu/ContextualMenu.tsx

```diff
@@ -0,0 +1,113 @@
+import React, { useState, Fragment } from "react";
+import styled from "styled-components";
+import CoreTokens from "../common/coreTokens";
+import MenuPropsType, { Item, Section as SectionType, BadgeProps as BadgePropsType } from "./types";
+import DxcBadge from "../badge/Badge";
+import MenuItem from "./MenuItem";
+
+const DxcContextualMenu = ({ items, defaultSelectedItemIndex = -1 }: MenuPropsType) => {
+  const [selectedItemIndex, setSelectedItemIndex] = useState<number>(defaultSelectedItemIndex);
+
+  const renderSingleItem = (item: Item, index: number) => (
+    <Li key={`option-${index}`}>
+      <MenuItem
```

Comment:
I think that role="menuitem" must be placed at `Li` level rather than in the inner level.

Reply:
Agreed Enrique! I uploaded the changes.